engine/spverrors/definitions.go

```diff
@@ -134,6 +140,18 @@ var ErrGettingPKIFailed = models.SPVError{Message: "getting PKI for contact fail
 // ErrSaveContact is when saving new contact failed
 var ErrSaveContact = models.SPVError{Message: "adding contact failed", StatusCode: 400, Code: "error-contact-adding-contact-failed"}
 
+// ErrUpdateContact is when updating contact failed
+var ErrUpdateContact = models.SPVError{Message: "updating contact failed", StatusCode: 400, Code: "error-contact-updating-contact-failed"}
+
+// ErrDeleteContact is when deleting contact failed
+var ErrDeleteContact = models.SPVError{Message: "deleting contact failed", StatusCode: 400, Code: "error-contact-deleting-contact-failed"}
+
+// ErrRejectContact is when rejecting contact failed
+var ErrRejectContact = models.SPVError{Message: "rejecting contact failed", StatusCode: 400, Code: "error-contact-rejecting-contact-failed"}
+
+// ErrAcceptContact is when accepting contact failed
+var ErrAcceptContact = models.SPVError{Message: "accepting contact failed", StatusCode: 400, Code: "error-contact-accepting-contact-failed"}
+
```

Comment:
I'm not sure if those should be with `StatusCode` 500. 🤔

Reply:
ok, let keeps 500 for failed actions